src/core/include/openvino/core/model.hpp

```diff
@@ -24,9 +24,18 @@
 #include "openvino/runtime/tensor.hpp"
 
 namespace ov {
+class Model;
+void copy_shared_objects(ov::Model& dst, const ov::Model& src);
```

Comment:
This function shouldn't be called from user-side (not exported) but is declared in ov namespace and user still may call it in the code by mistake. As I see in the code below, it is called from ov::clone_model. Can we just make ov::clone_model a friend of Model class and move copy_shared_objects to private Model class part?

Reply:
Ok, fixed